content/en/opentelemetry/collector_exporter/kafka_metrics.md

```diff
@@ -0,0 +1,349 @@
+-Dotel.metrics.exporter=otlp \
+-Dotel.exporter.otlp.endpoint=http://localhost:4317 \
+-jar /path/to/opentelemetry-jmx-metrics.jar
+
+// Kafka Producer
+java -jar -Dotel.jmx.service.url=service:jmx:rmi:///jndi/rmi://{KAFKA_PRODUCER_JMX_ADDRESS}/jmxrmi \ -Dotel.jmx.target.system=kafka-producer \
+-Dotel.metrics.exporter=otlp \
+-Dotel.exporter.otlp.endpoint=http://localhost:4317 \
+-jar /path/to/opentelemetry-jmx-metrics.jar
+
+// Kafka Consumer
+java -jar -Dotel.jmx.service.url=service:jmx:rmi:///jndi/rmi://{KAFKA_CONSUMER_JMX_ADDRESS}/jmxrmi \ -Dotel.jmx.target.system=kafka-consumer \
+-Dotel.metrics.exporter=otlp \
+-Dotel.exporter.otlp.endpoint=http://localhost:4317 \
+-jar /path/to/opentelemetry-jmx-metrics.jar
+```
+
+[1]: https://github.com/open-telemetry/opentelemetry-java-contrib/releases
+
+{{% /tab %}}
+
+{{% tab "Kubernetes" %}}
+
+The JMX Metric Gatherer is intended to be run as an uber jar and configured with properties from the command line. 
+
+In order to deploy this in kubernetes, you will need to build an image which contains JRE and the JMX Metrics Gatherer Jar. Please refer to the Dockerfile below for an example image that contains JRE and JMX Metrics Gatherer Jar.
+
+Dockerfile:
+```Dockerfile
+FROM alpine:latest as prep
+
+# JMX Metrics Gatherer Jar
+ARG JMX_GATHERER_JAR_VERSION=1.27.0
+ADD https://github.com/open-telemetry/opentelemetry-java-contrib/releases/download/v${JMX_GATHERER_JAR_VERSION}/opentelemetry-jmx-metrics.jar /opt/opentelemetry-jmx-metrics.jar
+# nonroot user id (https://groups.google.com/g/distroless-users/c/-DpzCr7xRDY/m/eQqJmJroCgAJ)
+ARG USER_UID=65532
+RUN chown ${USER_UID} /opt/opentelemetry-jmx-metrics.jar
+
+FROM gcr.io/distroless/java11-debian11:nonroot
+
+COPY --from=prep /opt/opentelemetry-jmx-metrics.jar /opt/opentelemetry-jmx-metrics.jar
+
+EXPOSE 4317 55680 55679
+ENTRYPOINT ["java"]
+CMD ["-Dotel.jmx.service.url=service:jmx:rmi:///jndi/rmi://kafka:1099/jmxrmi", \
+"-Dotel.jmx.target.system=kafka,jvm", \
+"-Dotel.metrics.exporter=otlp", \
+"-Dotel.exporter.otlp.endpoint=http://otelcol:4317", \
+"-jar", \
+"/opt/opentelemetry-jmx-metrics.jar"]
+```
+{{% /tab %}}
+
+{{< /tabs >}}
+
+## Log Collection
+
+See [Log Collection][4] for instructions on how to collect logs using the OpenTelemetry Collector.
+
+In order to appear in the OOTB Kafka Dashboard, the kafka logs need to be tagged with `source:kafka`. In order to do so, you can use an attributes processor:
+
+```yaml
+processors:
+  attributes:
+    actions:
+      - key: ddtags
+        value: "source:kafka"
+        action: insert
+```
+
+
+## Data collected
+
+| OpenTelemetry Metric | DD Metric | Source | Transform done DD Metric |
+|---|---|---|---|
+| otel.kafka.producer.request-rate | kafka.producer.request_rate | JMX Receiver / JMX Metrics Gatherer {target_system:kafka-producer} | |
+| otel.kafka.producer.response-rate | kafka.producer.response_rate | JMX Receiver / JMX Metrics Gatherer {target_system:kafka-producer} | |
```

Comment:
Should we move this to metrics mapping page?

Reply:
Moved.